specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json

```diff
@@ -3970,12 +4586,12 @@
         "x-ms-long-running-operation": true
       },
       "get": {
-        "operationId": "RestorePoints_Get",
-        "description": "The operation to get the restore point.",
+        "tags": [
+          "CapacityReservations"
+        ],
+        "operationId": "CapacityReservations_Get",
+        "description": "Retrieves information about a capacity reservation.",
```

Comment:
The operation that retrieves information about the capacity reservation.

Reply:
updated